### PR TITLE
Add slider debounce

### PR DESCRIPTION
**Pull request by Juniormunk**
Sliders for exposure and brightness would spam messages on the backend. This used to cause crashes and can cause it to get quite laggy / delayed. This will add a 20ms debounce which won't send the value to the backend until the value hasn't changed for 20ms.

### CODE REVIEW

File: photon-client/src/components/common/pv-slider.vue [hunk 1]
```diff
@@ -25,9 +25,23 @@ const emit = defineEmits<{
   (e: "input", value: number): void;
 }>();
 
+// Debounce function
+function debounce(func: (...args: any[]) => void, wait: number) {
+  let timeout: ReturnType<typeof setTimeout>;
```
Comment (mcm001) [inline photon-client/src/components/common/pv-slider.vue:30]: what is this initialized to? not sure what clearTimeout(timeout); does for something that's uninitialized?

**Review by mcm001 — APPROVED** [review]
seems to work

File: photon-client/src/components/common/pv-slider.vue [hunk 2]
```diff
@@ -25,9 +25,23 @@ const emit = defineEmits<{
...
+  return function (...args: any[]) {
+    clearTimeout(timeout);
+    timeout = setTimeout(() => func.apply(this, args), wait);
+  };
+}
+
+const debouncedEmit = debounce((v: number) => {
+  emit("input", v);
+  console.log("debouncedEmit", v);
```
Comment (mcm001) [inline photon-client/src/components/common/pv-slider.vue:39]: remove log

**Review by mcm001 — CHANGES_REQUESTED** [review]

File: photon-client/src/components/common/pv-slider.vue (hunk 2 above)
Comment (Juniormunk) [reply photon-client/src/components/common/pv-slider.vue:39]: Ope

**Comment by mcm001** [comment]
I'm also down to loosen up 20ms, that's pretty fast still?

**Review by mcm001 — APPROVED** [review]
Qual by test (it works on my machine). Leaving the time out number itself up to your engineering judgement.

**Pull request merged**